arroyo/examples/transform_and_produce/script.py

```diff
@@ -0,0 +1,58 @@
+import os
+from arroyo.backends.kafka.configuration import (
+    build_kafka_configuration,
+    build_kafka_consumer_configuration,
+)
+from arroyo.backends.kafka.consumer import (
+    KafkaConsumer,
+    KafkaProducer,
+)
+from arroyo.processing.processor import StreamProcessor
+
+
+from arroyo.types import Topic
+
+from arroyo.examples.transform_and_produce.factory import (
+    HashPasswordAndProduceStrategyFactory,
+)
+
+
+RAW_TOPIC = Topic("raw-topic")
+HASHED_TOPIC = Topic("hashed-topic")
+
+# Where the broker is, localhost:9092 when running locally
+BOOTSTRAP_SERVERS = os.environ.get("BOOTSTRAP_SERVERS") or ["localhost:9092"]
```

Comment:
How does this work if set via environment variable? Does bootstrap servers have to be an array?

Reply:
Technically `bootstrap_servers` has the type `Sequence[str]` which tells me it should be a list, tuple etc.
https://github.com/getsentry/arroyo/blob/3d83a4bed1b82e8a1d3b2c376fa23c1721f63606/arroyo/backends/kafka/configuration.py#L43
It just works if I pass a  `str` anyway, so a basic string as an environment variable should work. I put the default in a list to respect the type but I can change it if that doesn't make sense.

Reply:
If it's supposed to be a list, why not just make it a list? Even though it might just happen to work, I don't think we should rely on that.